code/client/index.html

```diff
@@ -126,7 +126,7 @@
 	<link href="/vendors/owl-carousel/owl.carousel.css" rel="stylesheet" type="text/css" />
 	<link href="/vendors/owl-carousel/owl.theme.css" rel="stylesheet" type="text/css" />
 </head>
-<body class="design-v2" onload="ApplyDesignExperiemnt();">
+<body onload="ApplyDesignExperiemnt();">
```

Comment:
Will this still allow the experiment to run, it will just default to no-experiment on-load and then snap to v2 once the experiment choice is fetched from Google?